recipes/somoclu/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "somoclu" %}
+{% set version = "1.7.4" %}
+{% set sha256 = "6182b688d9e58bd692d88f1d7edc9f4b536fba722417bec43b4b38eaebd13af9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-python-{{ version }}.tar.gz
+  url:  https://github.com/peterwittek/somoclu/releases/download/1.7.4/somoclu-python-1.7.4.tar.gz
```

Comment:
This will also break for new versions. The python version is [on pypi](https://pypi.python.org/pypi/somoclu), so you should be able to use the same source URL as for the other recipe:
`https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`.